tests/whist/server/api/game/test_action.py

```diff
@@ -7,6 +7,18 @@
 from whist.server.services.game_db_service import GameDatabaseService
 
 
+class PlayerNotReadyError(Exception):
+    """
+    Will check to see if player is ready before unreadying
+    """
+
+
+class GameNotFoundError(Exception):
+    """
+    Checks to see if user enters correct game_id
+    """
+
+
```

Comment:
```suggestion
```

@abant07  I suspect the error is that you redeclare the errors here, but you need to import them here.